invenio_communities/assets/semantic-ui/js/invenio_communities/community/featured-communities/FeaturedCommunities.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import { withCancel } from "react-invenio-forms";
+import { http } from "../../api/config";
+import { Grid, Message, Container, Loader } from "semantic-ui-react";
+import FeaturedCommunity from "./FeaturedCommunity";
+import PropTypes from "prop-types";
+import { i18next } from "@translations/invenio_communities/i18next";
+
+export default class FeaturedCommunities extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: undefined,
+      error: undefined,
+      isLoading: false,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  componentWillUnmount() {
+    this.cancellableFetch && this.cancellableFetch.cancel();
+  }
+
+  fetchData = async () => {
+    this.setState({ isLoading: true });
+
+    this.cancellableFetch = withCancel(http.get("/api/communities/featured"));
```

Comment:
```suggestion
    cancellableFetch = withCancel(http.get("/api/communities/featured"));
```
assignment to object instance, not needed

Reply:
Why not?
I think it's needed to call it in the fucntion `componentWillUnmount`? Am I missing something?